<ui version="4.0" >
 <class>ExtensionSystem::Internal::PluginErrorView</class>
 <widget class="QWidget" name="ExtensionSystem::Internal::PluginErrorView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>342</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>State:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="state" >
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Error message:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <widget class="QTextEdit" name="errorString" >
     <property name="tabChangesFocus" >
      <bool>true</bool>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
